Modules/HR/Config/config.php

```diff
@@ -3,7 +3,8 @@
 return [
     'name' => 'HR',
     'applicationEvaluation' => [
-        'cutoffScore' => 6,
+        'cutoffScore-resume-screening' => 6,
+        'cutoffScore-telephonic-interview' => 5,
```

Comment:
Let's follow one convention only
```suggestion
        'cut-off-score-resume-screening' => 6,
        'cut-off-score-telephonic-interview' => 5,
```

Please fix this at other places as well